tests/test_harvest.py

```diff
@@ -455,3 +456,40 @@ def test_oai_to_s3_harvest(self, mock_harvest, mock_prefix, mock_process, **kwar
         self.assertTrue(mock_harvest.called)
         self.assertTrue(mock_prefix.called)
         self.assertTrue(mock_process.called)
+
+
+    @mock.patch("tulflow.harvest.harvest_oai")
```

Comment:
Should lines 461-463 be indented?

Reply:
@sensei100 Nope, those are python decorators that need to be lined up with the function definition, as per the examples in https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch